### ISSUE TITLE
Add a statsd helper.

### ISSUE THREAD

**Issue by Natim** [mozilla-services/cliquet#480]
For now, it is quite verbose to use the statsd client:

```python
statsd = request.registry.statsd
if statsd:
    statsd.count("syncclient.status_code.200")
```

In https://github.com/mozilla-services/syncto/pull/49 @ametaireau suggested that we add a helper that let us do that:

```python
statsd(request, "count", "syncclient.status_code.200")
```

It could be good to also replace the use of this helper anywhere it is needed.

**Comment by greeshmab** [comment]
the helper function takes request and the status code as parameters.

**Comment by leplatrem** [comment]
> the helper function takes request and the status code as parameters. 

From what I read in the original description, it takes any kind of string. 

As a first version, I would suggest the helper to be `statsd_count` for example.

The count is called with any string passed:

```python
statsd_count(request, 'record-created')
statsd_count(request, 'error.found')
``` 

In a second iteration, if it is relevant (not clear to me to be honest), it could be interesting to have a single helper that receives the method name (`count`, `timer`, ...) as a parameter:

```python
statsd(request, 'count', 'record-created')
statsd(request, 'count', 'error.found')

with statsd(request, 'timer', 'computation.prime.numbers'):
    compute_prime_number()
```

**Comment by leplatrem** [comment]
Fixed with #574
